File: lihua-vue/src/api/monitor/server/type/jvm-monitor.ts
export interface JvmMonitor {
    /** java名称 */
    name: string;

    /** 版本 */
    version: string;

    /** 供应商 */
    vendor: string;

    /** 启动时间 */
    startTime: string;

    /** 运行时长 */
    runningTime: string;

    /** 运行参数 */
    inputArguments: string[];
}

File: lihua-biz/lihua-monitor/src/main/java/com/lihua/monitor/model/JvmMonitor.java
package com.lihua.monitor.model;

import lombok.Data;
import lombok.experimental.Accessors;

import java.time.LocalDateTime;
import java.util.List;

@Data
@Accessors(chain = true)
public class JvmMonitor {

    /**
     * java名称
     */
    private String name;

    /**
     * 版本
     */
    private String version;

    /**
     * 供应商
     */
    private String vendor;

    /**
     * 启动时间
     */
    private LocalDateTime startTime;

    /**
     * 运行时长
     */
    private String RunningTime;

    /**
     * 运行参数
     */
    private List<String> inputArguments;
}
